Jenkinsfile

```diff
@@ -1,8 +1,6 @@
 /*
- * Hibernate Validator, declare and validate application constraints
- *
- * License: Apache License, Version 2.0
- * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright Red Hat Inc. and Hibernate Authors
```

Comment:
Come on... the *main* goal of this tagline was to unify it across projects so that we -- the Hibernate team -- don't need to use separate IDE config for each project, at least when it comes to headers.

Please, don't apply this suggestion :(

Reply:
Well, my problem is that "Hibernate authors" is not a thing. It might be practical for you but it's really artificial given `Hibernate` is not a thing really as far as `authors` go. It's an umbrella for various projects.

`Copyright Red Hat Inc. and other contributors` or something like that would make more sense to me.

Anyway, do whatever is best for you. I'll go share my insights elsewhere :).